Objects/typeobject.c

```diff
@@ -45,7 +45,7 @@ class object "PyObject *" "&PyBaseObject_Type"
 
 // bpo-42745: next_version_tag remains shared by all interpreters because of static types
 // Used to set PyTypeObject.tp_version_tag
-static unsigned int next_version_tag = 0;
+static unsigned int next_version_tag = 1;
```

Comment:
Would it be worth it to change PyTypeObject.tp_version_tag type to uint64_t to reduce the risk of overflow?

2^64 version changes is less likely than 2^32 changes.

Reply:
PyType_ClearCache is documented to return `unsigned int`, not `uint_64t`, so I don't know if we can change that without deprecation. https://docs.python.org/3/c-api/type.html#c.PyType_ClearCache.

Reply:
64 bit versions would make the type cache twice the size and might well make performance worse.

Classes don't change that much. If it happens that long running programs use 2**32 versions, then it is likely that a small set of classes change often. In that case we could limit the number of version that an individual class can use up.